pravega/pravega-operator
# Issue 395: Add helm 3 support, post-install hooks and make cluster specs overridable through helm values

Signed-off-by: SrishT <Srishti.Thakkar@dell.com>

### Change log description
Updates the charts and the the helm commands to work with helm v3. It makes all the cluster specs overridable through Helm values. Also adds self validating post-install hooks to the charts.

### Purpose of the change
Fixes #395 #393 #304 #396 

### What the code does
Makes all cluster specs overridable and adds self-validating post-install hooks to the charts.

### How to verify it
User should be able to follow the instructions mentioned in the documentation to install/delete a pravega cluster using charts with helm v3. User should be able to modify any of the pravega cluster specs through the values.yaml file.
